ui/src/shared/components/views/Markdown.scss

```diff
@@ -3,32 +3,13 @@
   ------------------------------------------------------------------------------
 */
 
-.markdown-cell {
-  position: absolute !important;
-  top: 0;
-  left: 0;
-  border-radius: $ix-radius;
+.markdown-cell .cf-dapper-scrollbars--track-y {
+  background-color: $g2-kevlar !important;
 }
 
 .markdown-cell--contents {
-  padding: $ix-marg-b + $ix-marg-a 26px;
-  font-size: 13px;
-}
-
-.markdown-cell > .fancy-scroll--track-v,
-.markdown-cell > .fancy-scroll--track-h {
-  z-index: 2;
-}
-
-.markdown-cell:after {
-  pointer-events: none;
-  content: '';
-  position: absolute;
-  bottom: 0;
-  left: 0;
-  width: 100%;
-  @include gradient-v(rgba($g3-castle, 0), $g3-castle);
-  z-index: 1;
+  padding: $cf-marg-b + $cf-marg-a 26px;
+  font-size: 14px;
```

Comment:
The previous font-size was `13px`, and we also have other instances where the app had been using `13pt` font, should we transition the font-sizes from 13->14 / should we set variables to do that?

Reply:
The `14px` deviates but is much nicer to read. Maybe I could set the size using `em` or `rem` so at least it's based on the root font size instead of a fixed number